.travis.yml

```diff
@@ -54,6 +54,7 @@ script:
   - cargo doc --features "dox,embed-lgpl-docs"
   - cargo test --features "$FEATURES,png,pdf,svg,pdf,xcb"
   - cargo test --no-default-features --features "$FEATURES,png,pdf,svg,pdf,xcb"
+  - cargo test --features "$FEATURES xcb pdf svg png ps use_glib"
```

Comment:
Everywhere else we use commas, here you use spaces now :) And how is this different from `cargo test --features "$FEATURES,png,pdf,svg,pdf,xcb"` (`use_glib` is default feature)

Reply:
Woups, local test. Removing it.